docs/docs/overview/quickstart/get-started-with-angular.md

```diff
@@ -0,0 +1,419 @@
+
+![in-app-configuration.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/in-app-configuration_mmzx98.png)
+
+I’ll briefly explain the function of each label in the image above.
+
+- **1 - Preview**: Shows you a glimpse of how each notification item will look like in the Notification Center UI.
+- **2 - Avatar:** If turned on, each notification item will show the avatar of the subscriber.
+- 3 - **Action:** With this, you can add a primary and secondary call to action button to each notification item.
+- **4 - Notification Feeds:** This displays a stream of specific notifications. You can have multiple feeds to show specific notifications in multiple tabs.
+- **5 - Redirect URL** - This is the URL to which a subscriber can be directed when they click on a notification item.
+- **6 - Filter** - This feature allows you to configure the criteria for delivering notifications. For instance, you can apply a filter based on a subscriber's online status to send them an email if they were online within the last hour. Read [more about filters](https://docs.novu.co/platform/step-filter/#subscriber-seen--read-filters).
+- **Editor** - You can add text that you want displayed in each notification item. Additionally, you can specify custom variables using `{{ }}`. This means you can inject variables from your code into a notification item's text via a payload.
+
+1. Feel free to add only text for now and rename the notification template to `Onboarding In App`. It automatically creates a slug-like identifier that will be needed in later steps to trigger a notification.
+
+![Screenshot 2023-05-21 at 09.33.43.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776584/Screenshot_2023-05-21_at_09.33.43_ujx2td.png)
+
+![Screenshot 2023-05-21 at 09.20.51.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776585/Screenshot_2023-05-21_at_09.20.51_cueewk.png)
+
+Next, we’ll learn how to create subscribers on Novu - _Recipients of Notifications_
+
+## Create A Subscriber
+
+Click **Subscribers** on the left sidebar of the [Novu dashboard](https://web.novu.co/subscribers) to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign up.
+
+![subscribers.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776584/subscribers_ppir3w.png)
+
+Now, let's create a subscriber on Novu.
+
+Novu has a plethora of backend SDKs (Node.js, PHP, .NET, Go, Ruby, Python and Kotlin) to choose from to create a subscriber programmatically. This is the recommended method.
+
+<iframe width="800" height="450" src="https://codesandbox.io/p/sandbox/create-subscriber-m5uibf?file=%2Findex.js%3A1%2C1&embed=1" allowfullscreen></iframe>
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.identify('132', {
+  email: 'john.doe@domain.com',
+  firstName: 'John',
+  lastName: 'Doe',
+  phone: '+13603963366',
+});
+```
```

Comment:
Please use the multi-language component so that you can add the code for creating subscribers in PHP as well.

Reply:
Added.